### PR TITLE
patientlist backend connection

### LINKED ISSUE #509: PatientList Backend Connection

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #509

### CODE REVIEW

**Comment by DasProffi** [comment]
Blocked by https://github.com/helpwave/services/issues/341

File: tasks/components/layout/PatientList.tsx
Comment (MaxSchaefer) [inline tasks/components/layout/PatientList.tsx:1]: It's both, a component and a layout. Wouldn't it be better if the toggable/dropdown sections were a component?

File: tasks/components/layout/PatientList.tsx [hunk 1]
```diff
@@ -103,57 +111,87 @@ export const PatientList = ({
...
+                </button>
+              </div>
+              {openedSections.discharged && filteredDischarged.map(patient => (
+                <div key={patient.id} className={tw('flex flex-row pt-2 border-b-2 justify-between items-center')}>
+                  <Span className={tw('font-space font-bold')}>{patient.name}</Span>
+                  <Button color="negative" variant="textButton" onClick={() => {
+                    // TODO delete
+                  }}>
+                    {translation.delete}
+                  </Button>
```
Comment (MaxSchaefer) [inline tasks/components/layout/PatientList.tsx:188]: Just hide the button if the functionality is not implemented yet.

**Review by MaxSchaefer — DISMISSED** [review]

File: tasks/components/layout/PatientList.tsx
Comment (DasProffi) [reply tasks/components/layout/PatientList.tsx:1]: yes

**Review by MaxSchaefer — APPROVED** [review]

**Comment by MaxSchaefer** [comment]
🙌

**Pull request merged**